packages/loopring_v3/contracts/iface/IExchangeV3.sol

```diff
@@ -15,9 +15,10 @@
   limitations under the License.
 */
 pragma solidity ^0.5.11;
+pragma experimental ABIEncoderV2;
 
 import "../iface/IExchange.sol";
-
+import "./ExchangeData.sol";
```

Comment:
> should we expose getDepositContract in this interface?

Sure, will add it.

> should we still provide withdrawTokenNotOwnedByUsers and delegate the call to the DepositContract?

No, because this will be deposit contract dependent. It's up to the deposit contract to decide on what to do with these funds. It could allow it to be withdrawn but it could also use it differently. It may also be impossible to do it all because funds could be locked somewhere else so the deposit contract may not know how much funds it actually has.